app/gui/src/controller/graph/widget.rs

```diff
@@ -410,6 +410,10 @@ impl QueryData {
         Ast::raw_text_literal(arg).repr()
     }
 
+    fn as_unresolved_symbol(arg: &str) -> String {
+        ".".to_string() + arg
```

Comment:
It's not clear to me what "unresolved symbol" means in this context. The syntax also doesn't make sense to me initially, so it warrants some expanation as a code comment.

Also, it would be better to use formatting here: `format!(".{arg}")`. Probably doesn't even have to be a dedicated function call, and can be directly inlined into `visualization_metadata` (with comment).